specification/automation/resource-manager/Microsoft.Automation/stable/2023-11-01/examples/createOrUpdatePowerShell72Module.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "automationAccountName": "myAutomationAccount33",
+    "moduleName": "OmsCompositeResources",
+    "api-version": "2023-11-01",
+    "parameters": {
+      "properties": {
+        "contentLink": {
+          "uri": "https://teststorage.blob.core.windows.net/dsccomposite/OmsCompositeResources.zip",
+          "contentHash": {
+            "algorithm": "sha265",
+            "value": "07E108A962B81DD9C9BAA89BB47C0F6EE52B29E83758B07795E408D258B2B87A"
+          },
+          "version": "1.0.0.0"
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Automation/automationAccounts/myAutomationAccount33/powershell72modules/OmsCompositeResources",
+        "name": "OmsCompositeResources",
+        "type": "Microsoft.Automation/AutomationAccounts/PowerShell72Modules",
+        "location": "East US 2",
+        "tags": {},
+        "etag": null,
+        "properties": {
+          "isGlobal": false,
+          "version": null,
```

Comment:
>null [](http://example.com/codeflow?start=21&length=4)

the response property (version) is not going with the input request. Pls fix.

Reply:
Fixed.